configs/default/IFRC-JBF7_DJI.config

```diff
@@ -0,0 +1,175 @@
+# version
```

Comment:
Same here.

Reply:
Thank you for your reminder, it has been corrected.